ui/app/styles/core/charts.scss

```diff
@@ -2,22 +2,19 @@
   border: $light-border;
   border-radius: $radius-large;
   padding: $spacing-l $spacing-l $spacing-s $spacing-l;
+  margin-bottom: $spacing-m;
   height: 547px; // TODO amend for specific chart heights
   width: 100%;
-
-  // ARG TODO we can manage this using css grid
-  // > div.is-border {
-  //   border: 0.3px solid $ui-gray-200;
-  //   margin-bottom: $spacing-xxs;
-  // }
 }
-.single-chart {
+
```

Comment:
I reorganized this file so that all grid-layout stuff is first, then styling (i.e. font-size for <h2>'s, etc), THEN misc. items like tooltip, chart-specific styles (gridlines for the vertical chart, or heavy-weighted labels for horizontal chart)

i had trouble finding the styles when they were scattered throughout but let me know what's more intuitive for you!